### PR TITLE
tracker(dm): add UT and fix uncleaned folder

### LINKED ISSUE #4816: Remove schema tracker dir in ut
### Which jobs are flaking?

```
make dm_unit_test
```

### Jenkins logs or GitHub Actions link

```log
(paste logs here)
```

### Anything else we need to know

- Does this test exist for other branches as well?

- Has there been a high frequency of failure lately?

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve?

Issue Number: close #4816 

### What is changed and how it works?

- add tracker.Close
- also add a test case that SHOW CREATE TABLE in downstream returns placement rule comment (I get the output from RFC, not test it in real TiDB)
- move some files in sqlmodel package

### Check List 

Tests 

 - Unit test

Related changes

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Comment by lance6716** [comment]
ping @GMHDBJD @WizardXiao

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: dm/pkg/schema/tracker_test.go [hunk 1]
```diff
@@ -114,6 +116,8 @@ func (s *trackerSuite) TestTiDBAndSessionCfg(c *C) {
 	c.Assert(mock.ExpectationsWereMet(), IsNil)
 	err = tracker.Exec(context.Background(), "", "create database testdb;")
 	c.Assert(err, IsNil)
+	err = tracker.Close()
+	c.Assert(err, IsNil)
```
Comment (D3Hunter) [inline dm/pkg/schema/tracker_test.go:120]: maybe make this and below two into a `defer` after `newTracker` too

**Review by D3Hunter — APPROVED** [review]
rest lgtm

File: dm/pkg/schema/tracker_test.go (hunk 1 above)
Comment (lance6716) [reply dm/pkg/schema/tracker_test.go:120]: `tracker` will be lost by overwritten when defer closure call, I need to assign every tracker for every `NewTracker` then

Comment (D3Hunter) [reply dm/pkg/schema/tracker_test.go:120]: just in case `tracker.close` is not called for previous assert may failed

**Pull request merged**